src/server/accounting.c

```diff
@@ -1155,6 +1158,10 @@ account_jobend(job *pjob, char *used, int type)
 			patlist = pjob->ji_wattr[(int)JOB_ATR_resc_used].at_user_encoded;
 		else if (pjob->ji_wattr[(int)JOB_ATR_resc_used].at_priv_encoded != NULL)
 			patlist = pjob->ji_wattr[(int)JOB_ATR_resc_used].at_priv_encoded;
+		else 
+			encode_resc(&pjob->ji_wattr[(int)JOB_ATR_resc_used], 
+				&temp_head, (job_attr_def+(int)JOB_ATR_resc_used)->at_name, 
```

Comment:
Why are you using the form of job_attr_def+(int)JOB_ATR_resc_used instead of a direct index?  You directly index off of pjob.